### PR TITLE
Fix:Inconsistent Assigned Employee status

### LINKED ISSUE #203: Showing Null at place Assigned Employee in Customer Detail View
**Summary:** 

As you will open any customer details (Jone Doe) for no employee assigned then you can see the ```null```.  It should show ```Not Assigned```.

it is working fine for other but for this customer not. 

**Steps to reproduce:** 

Go to customer list and click on the customer (Jone Due) that does not have any assigned employee. you will see null.

**Expected behavior:** 

It should have text ```Not Assigned``` instead of ```null```

**Observed behavior:** 
showing ```null``` instead of ```Not Assigned```

**Device and Android version:** 

Moto G turbo

**Screenshots:** 

image
image

### PR DESCRIPTION
**Pull request by AkshGautam**
Please make sure these boxes are checked before submitting your pull request - thanks!

Fixes #203

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
screenshot_2018-01-17-15-28-15
@therajanmaurya  Please review.Thanks.

File: app/src/main/java/org/apache/fineract/ui/online/customers/customerdetails/CustomerDetailsFragment.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ public void showCustomerDetails(Customer customer) {
 
         String title = customer.getGivenName() + " " + customer.getSurname();
         String subtitle;
-        if (customer.getAssignedOffice() == null) {
+        if (customer.getAssignedEmployee() == null) {
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/fineract/ui/online/customers/customerdetails/CustomerDetailsFragment.java]: Good work 👍

**Comment by therajanmaurya** [comment]
image
image
@AkshGautam Please make sure to mention the issue number in PR description for the next time, So that I don't have to go and find the issue to close.

**Comment by AkshGautam** [comment]
@therajanmaurya sorry sir for the inconvenience caused.Would take care from next time.

**Pull request merged**